tests/serializers/test_data.py

```diff
@@ -477,13 +432,76 @@ def serializerTest(self, format):
 
     # Assert that the deserialized data is the same
     # as the original source
-    for func, pk, klass, datum in test_data:
-        func[1](self, pk, klass, datum)
+    for func, pk, klass, datum in data:
+        with self.subTest(klass=klass, data=datum):
+            func[1](self, pk, klass, datum)
 
     # Assert that the number of objects deserialized is the
     # same as the number that was serialized.
     for klass, count in instance_count.items():
```

Comment:
Am I missing something, or is `instance_count` always an empty dictionary here? I never realized this before. It may be out of scope in this review.

Reply:
That's how I'm reading it as well. I did some digging and it looks like 12716794dbcf2e1bf7a07bd18dfc0ae24be4e6f8 is the commit that added `instance_count` (the filename is different but those tests have moved around a few times). Seems like the bug has always been there.

I changed the `for klass in instance_count:` above (line 465 as of now) to `for klass in {klass for _, _, klass, _ in data}:` (I'm guessing that was the intent of the original commit) and the tests still seem happy.

Reply:
Great!